### DISCUSSION TITLE
Porous media module: to output final permeability field from different reads

### DISCUSSION THREAD

**Discussion by zhang-rite** [idaholab/moose#17253] [Q&A Modules: General] [answered]
Hi 

I set the permeability of different regions (e.g., block1,2...) in a model separately. 
So these can be saved in the exodus file separately.
But how can I export the final permeability field of the entire model to an exodus file by one variable name for visualization?
Perhaps also for the setting of other materials.
Thank you.

**Comment by WilkAndy** [comment, marked as answer]
Just create an AuxVariable that's defined over all the blocks

**Reply by zhang-rite** [reply]
Andy, Thanks always. The problem is I have defined several Auxvarible over individual blocks. 
Each is set via **RandomIC**. I want to apply random permeability fields with different ranges.

**Reply by WilkAndy** [reply]
You can probably join the variables together in paraview

Alternatively, define a new AuxVariable, and use [PorousFlowPropertyAux](https://mooseframework.inl.gov/source/auxkernels/PorousFlowPropertyAux.html) with `property = permeability`, along with `row` and `column` to populate it.
